index.md

```diff
@@ -37,6 +37,30 @@ consumers of your library:
 A scope may be provided to a commit's type, to provide additional contextual information and
 is contained within parenthesis, e.g., `feat(parser): add ability to parse arrays`.
 
+## Examples
+
+### Commit message with body and breaking change
```

Comment:
`### Commit message with body and breaking change`

Should be:

`### Commit message with description and breaking change in footer`